src/main/java/org/deepsymmetry/beatlink/DeviceFinder.java

```diff
@@ -52,30 +52,14 @@ public class DeviceFinder extends LifecycleParticipant {
      */
     private static final AtomicLong firstDeviceTime = new AtomicLong(0);
 
-    /**
```

Comment:
Whoops, it looks like you must have a merge conflict and accidentally deleted code that I intend to have in here?